src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -666,6 +691,11 @@ abstract class GCMEngine {
         byte[] originalOut = null;
         int originalOutOfs = 0;
 
+        // True if op is in-place array decryption with the input & output
```

Comment:
// Setting `inPlaceArray` to true turns off combined intrinsic processing.

Reply:
yeah that's better

Reply:
Actually the replacement isn't entirely accurate.  This only applies to decryption and for buffers that don't overlap where input is ahead of output.  That's why the comment is so wordy